src/Command/Site/StatisticsCommand.php

```diff
@@ -115,7 +116,12 @@ private function getNodeTypeCount($nodeType)
      */
     private function getCommentCount()
     {
-        $entityQuery = $this->entityQuery->get('comment')->count();
+        try {
+            $entityQuery = $this->entityQuery->get('comment')->count();
+        } catch (PluginNotFoundException $e) {
+            return 0;
```

Comment:
But it is not the value CLI application returns, but the value of comments which is 0 when "Comment" module is not installed. Btw I agree, that it would be better to not catch exception but prevent it. I can replace catch block with something like this:

```php
if (!\Drupal::moduleHandler()->moduleExists('comment')) {
    return 0;
}
```
What do you think about such changes?